webpack.config.js

```diff
@@ -0,0 +1,76 @@
+const devCerts = require("office-addin-dev-certs");
+const { CleanWebpackPlugin } = require("clean-webpack-plugin");
+const CopyWebpackPlugin = require("copy-webpack-plugin");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+const webpack = require("webpack");
+
+module.exports = async (env, options) => {
+    const dev = options.mode === "development";
+    const config = {
+        devtool: "source-map",
+        entry: {
+            commands: "./src/commands/commands.ts",
+            fallbackauthtaskpane: "./src/taskpane/fallbackauthtaskpane.ts",
+            polyfill: "@babel/polyfill",
+            taskpane: "./src/taskpane/taskpane.ts",
+        },
+        resolve: {
+            extensions: [".ts", ".tsx", ".html", ".js"]
```

Comment:
Are we supporting tsx here because this template could be used with React or with a non-React option in yo office?

Reply:
I believe so. I will let @TCourtneyOwen confirm